src/logic/documents/scorecards.js

```diff
@@ -366,18 +377,21 @@ const scorecard = ({
       alignment: 'center',
     },
     {
+      fontSize: 11,
```

Comment:
We should make the font size consistent here and in the scr/result/judge/comp row. Also, why do we need to change it?